content/docs/cml/get-started-with-cml-on-gitlab.md

```diff
@@ -0,0 +1,109 @@
+# Get Started with CML on GitLab
+
+Here, we'll walk through a tutorial to start using CML on GitLab. For
+simplicity, we'll show the demo in GitLab Actions, but these instructions are
+valid for all supported Continuous Integration(CI) systems (with exceptions as
+noted!).
+
+1. Fork our
+   [example project repository](https://gitlab.com/iterative.ai/example_cml).
+   Click on Fork and select what namespace to fork the project to.
+
+   ![](/img/gitlab_fork_cml_project.png)
```

Comment:
I didn't read any of this yet 😬 but I just noticed that most of images are blurry (and some have strange edges):

![image](https://user-images.githubusercontent.com/1477535/107546341-de5d0480-6b91-11eb-8137-a20deacc3e97.png)

Reply:
Ya.  In my first pass I noted this.  Need to figure out how to make them better resolution.

Reply:
![image](https://user-images.githubusercontent.com/34289199/107566709-0c464700-6b9a-11eb-984c-e9f36f0867f5.png)

Reply:
So I went back to work on the pics for the instructions so they are not blurry. So I had to start all over otherwise I couldn't get to the create merge request picture.  Gitlab has moved where you go to set up the personal access token from where I set it up originally.  I'm trying to figure out so I can update that part of the instructions too....

Reply:
> figure out how to make them better resolution

I remember on Mac you can zoom into the screen at any point (3 finger pinch was it?) so using a high display resolution, try to zoom in to the desired content and then [take a screenshot](https://support.apple.com/en-asia/guide/mac-help/mh26782/mac). I think it should already save the file as .png by default.

Reply:
Is it addressed? If it's fixed @jendefig you can click the `Resolve converstaion` button below